INSTALL.md

```diff
@@ -41,7 +38,7 @@ are checked at configure time, preventing compilation. In the,
 unlikely, event an incompatibility remains undetected, using Floats
 would enable to prove False on this architecture.
 
-Note that `num` and `lablgtk3-sourceview3` should be properly
+Note that `lablgtk3-sourceview3` should be properly
```

Comment:
Shouldn't `zarith` be mentioned on this line?

Reply:
Tweak the whole paragraph a bit.